wapitiCore/attack/mod_brute_login_form.py

```diff
@@ -0,0 +1,184 @@
+from itertools import chain
+from os.path import join as path_join
+import requests
+from bs4 import BeautifulSoup
+from requests.exceptions import ReadTimeout
+from wapitiCore.net.web import Request
+from wapitiCore.attack.attack import Attack
+from wapitiCore.language.vulnerability import Vulnerability, Anomaly, _
+from wapitiCore.net import web
+
+
+class mod_brute_login_form(Attack):
+    time_to_sleep = ''
+    name = "brute_login_form"
+    payloads_username = []
+    payloads_password = []
+    PAYLOADS_FILE = "passwords.txt"
+    PAYLOADS_FILE_USER = "users.txt"
+    PAYLOADS_SUCCESS = "successMessage.txt"
+    PAYLOADS_FAIL = "incorrectMessage.txt"
+    MSG_VULN = _("Brute force attack success")
+    current_request_url = None
+    current_request_method = None
+    password_parameter = []
+    username_parameter_field = []
+    submit_var_name = None
+    submit_var_value = None
+
+    def set_timeout(self, timeout):
+        self.time_to_sleep = str(1 + int(timeout))
+
+    def has_password_field(self):
+        list = ['email','user','mail']
+        content = requests.get(self.current_request_url).content
```

Comment:
Use self.crawler.send instead of requests

Reply:
fixed